.changeset/config.json

```diff
@@ -0,0 +1,18 @@
+{
+  "$schema": "https://unpkg.com/@changesets/config@2.0.0/schema.json",
+  "changelog": [
+    "@changesets/changelog-github",
+    { "repo": "FuelLabs/fuel-ts" }
```

Comment:
This here should be `FuelLabs/fuel-wallet`?

Reply:
✅